sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -203,9 +203,9 @@ def get_transceiver_bulk_status(self):
             bulk_status["tx%dpower" % i] = self.mw_to_dbm(tx_power[i - 1]) if self.get_tx_power_support() else 'N/A'
 
         laser_temp_dict = self.get_laser_temperature()
-        bulk_status['laser_temperature'] = laser_temp_dict['monitor value']
         self.vdm_dict = self.get_vdm()
         try:
+            bulk_status['laser_temperature'] = laser_temp_dict['monitor value']
```

Comment:
not sure I understand, I moved `bulk_status['laser_temperature'] = laser_temp_dict['monitor value']` to `try` section to avoid the case that `laser_temp_dict` is none